docs/getting-started.md

```diff
@@ -1,276 +1,136 @@
-# Getting Started
+# Getting Started with SnapTrade
 
-Welcome to the SnapTrade developer hub. You'll find comprehensive documentation for all our endpoints, as well as detailed guides to help you in your build.
+## Introduction
 
-If you need further assistance or have any questions, please send us an email at api@snaptrade.com.
+Welcome to SnapTrade! This is intended to be one of the first documents you read when learning how to use the SnapTrade API and will answer many questions you might have while getting started. This document when read from top-to-bottom is also a guide for making your first connection, and will get you set up to start pulling account data and placing trades.
 
-If you already have API Keys, you can use [our demo
-tool](demo:) to start making requests.
+Please see the FAQ of each section, as any questions you might have should be answered there.
 
----
+If after reading this document you still have questions or need help, please do not hesitate to contact us at [support@snaptrade.com](mailto:support@snaptrade.com) or through <https://discord.gg/UDwzZUuQ>.
 
-## API Status
+## API Keys
 
-Call the :api[ApiStatus_check] endpoint to make sure the API is live and that you can make the most basic request. You should receive a response indicating the API status, current server timestamp, and internal API minor version number.
+![image.png](Getting%20Started%20with%20SnapTrade%201abfeaa69a1c808d9d92d671e5a5c95e/image.png)
 
-Example:
+Your `API key` consists of a `clientId` and `consumerKey`. Please note that your `consumerKey` is sensitive information and should be kept secure at all times. This `API key` is used to make any requests to the SnapTrade API, and is the first thing required to get started.
 
-```json
-{ "version": 151, "timestamp": "2022-01-24T19:02:46.440074Z", "online": true }
-```
+### How to: Create a new API Key
 
----
+When you are ready to get started with SnapTrade and want to start making requests to our API, you will need to create your free and paid `API keys`. You may create only your free key if preferred.
 
-## Developer SDKs
+1. Create a SnapTrade account at <https://dashboard.snaptrade.com/home>.
+2. Verify your email (You cannot create any `API Keys` until your account is verified).
+3. Generate your free `API Key` through <https://dashboard.snaptrade.com/api-key>.
+4. Add a payment method through the Settings>Billing page of <https://dashboard.snaptrade.com/settings/billing>.
+5. Upgrade to Pay-as-you-Go through Settings>Billing page of <https://dashboard.snaptrade.com/settings/billing>.
+6. Generate your paid `API Key` through <https://dashboard.snaptrade.com/api-key>.
```

Comment:
Probably should remove these, if the goal is help someone get started ASAP they're probably only interested in testing out the water with a free key first

Reply:
If we want to mention the paid key, we can create a separate guide "Productionize your app" or something similar and link to it from here.

Reply:
I've noted these steps as optional for now, but when there is a guide for going to prod - I will revisit this